horizon/static/framework/framework.module.js

```diff
@@ -126,11 +126,11 @@
   function httpRedirectLogin($q, $rootScope, $window, frameworkEvents, toastService) {
     return {
       responseError: function (error) {
-        if (error.status === 401) {
+        if (error.status === 401 && !error.config.ignore401) {
```

Comment:
Do you ever use this `ignore401` flag? Perhaps it should be like `ignoreClientErrors` and be one flag that ignores any 4xx response code.

Reply:
Another idea is `redirectOnAuthErrors` but invert the boolean logic. See below for example.

Reply:
`ignore401 ` is not used. I was just trying to match 401 with 403. I'll update to invert the boolean logic.